dm/master/server.go

```diff
@@ -1027,17 +1027,47 @@ func parseAndAdjustSourceConfig(contents []string) ([]*config.SourceConfig, erro
 			return cfgs, err
 		}
 		if err = cfg.Adjust(fromDB.DB); err != nil {
+			fromDB.Close()
 			return cfgs, err
 		}
 		if _, err = cfg.Yaml(); err != nil {
+			fromDB.Close()
 			return cfgs, err
 		}
 
+		fromDB.Close()
 		cfgs[i] = cfg
 	}
 	return cfgs, nil
 }
 
+func adjustTargetDB(ctx context.Context, dbConfig *config.DBConfig) error {
```

Comment:
How about putting this function into `task.go` as a member of `TaskConfig`? we may need to use it in other places later.

Reply:
circle import if move it to task.go because we import config in pkg.conn. 🤔